cockpit/handlers/camera.py

```diff
@@ -130,6 +130,22 @@ def __init__(self, name, groupName, callbacks, exposureMode,
                     "%s imager" % name, "imager",
                     {'takeImage': softTrigger}))
 
+        #subscribe to save and load setting calls to enabvle saving and
+        #loading of configurations.
+        events.subscribe('save exposure settings', self.onSaveSettings)
+        events.subscribe('load exposure settings', self.onLoadSettings)
+
+
+    ## Save our settings in the provided dict.
+    def onSaveSettings(self, settings):
+        settings[self.name] = self.getIsEnabled()
+
+    ## Load our settings from the provided dict.
+    def onLoadSettings(self, settings):
+        if self.name in settings:
+            # only chnage state if we need to as this is slow
+            if (self.getIsEnabled() != settings[self.name]):
```

Comment:
parentheses around this condition which is a one line comparison are not needed.

Reply:
ok, but it made it clearer to me what was being done. Please feel free to remove them in each case if you want.